content/docs/1.5.0/deploy/install/_index.md

```diff
@@ -99,6 +99,10 @@ Below are the directories Longhorn components requiring access with root and pri
   - /var/lib/longhorn/engine-binaries: The default path for storing the Longhorn engine binaries.
 - Longhorn Instance Manager (Replica)
   - /: Access any data path on this node and access Longhorn engine binaries.
+- Longhorn Instance Manager (AllInOne)
```

Comment:
Please just delete `Longhorn Instance Manager (Replica/Engine)` directly. 

Also, no need to mention `(allinone)`